.gitignore

```diff
@@ -75,3 +75,6 @@ ssh.config
 /tctl
 /teleport
 /tsh
+
+# Go workspace file
+go.work
```

Comment:
```suggestion
# Go workspace files
go.work
go.work.sum
```
(golang/go#51941)
I'm getting a non-empty `go.work.sum` on current master after a `go work init && go work use -r . && go mod tidy`, which [can occasionally happen](https://go.googlesource.com/proposal/+/master/design/45713-workspace.md#files) so it's worth ignoring as well.